src/Hedgehog/Linq/Property.fs

```diff
@@ -162,7 +162,8 @@ type PropertyExtensions private () =
 
     [<Extension>]
     static member Select (property : Property<'T>, mapper : Func<'T, 'TResult>) : Property<'TResult> =
-        Property.map mapper.Invoke property
+        property
+        |> Property.bind (Property.ofThrowing mapper.Invoke)
```

Comment:
Shouldn't this be how `map` works? I like having the LINQ interface just forward calls, and not having different functionality.

Reply:
I would also prefer if each C# method could just forward to its corresponding F# function.  However, that approach doesn't pass the test that I added.  We need to compensate for the fact that CEs in F# are not the same as query syntax in C#.  I don't care much how that is achieved, but I think @altxt is correct that the current behavior is wrong.